usvm-ts/src/main/kotlin/org/usvm/TSExprResolver.kt

```diff
@@ -182,6 +184,14 @@ class TSExprResolver(
         TODO("Not yet implemented")
     }
 
+    override fun visit(expr: EtsAwaitExpr): UExpr<out USort>? {
+        TODO("Not yet implemented")
+    }
+
+    override fun visit(expr: EtsYieldExpr): UExpr<out USort>? {
+        TODO("Not yet implemented")
+    }
```

Comment:
I don't now if anyone actually uses `usvm-ts`, so I can't say if it's ok to leave it in an unimplemented state

Reply:
We have no usages of usvm-ts. Moreover, usvm-ts is in outdated state wrt jacodb and will be updated/rewritten later.